contracts/mocks/PriceFeedMock.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.7;
+
+import { IPriceFeed } from "../external/DappHub/interfaces/IPriceFeed.sol";
+
+// Mock contract implementation of PriceFeed functions
```

Comment:
Why are we playing with the Maker PriceFeeds? We only query their Medianizers.

Reply:
PriceFeedsMock return uint256 vs their contracts do bytes32. So its incompatible with what our contract expects